changelogs/fragments/3113-support-for-session-operations.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Support for Redfish session operations for Ansible playbooks.
```

Comment:
I *think* this is already supported, see: https://github.com/ansible-collections/community.general/blob/e83838380229c44ad8a72bdccd9577753c4f71d6/plugins/module_utils/redfish_utils.py#L56-L65

Reply:
That's a good catch... I had not noticed that before...